packages/kbn-dev-utils/src/index.ts

```diff
@@ -21,5 +21,5 @@ export { withProcRunner } from './proc_runner';
 export { ToolingLog, ToolingLogTextWriter, pickLevelFromFlags } from './tooling_log';
 export { createAbsolutePathSerializer } from './serializers';
 export { CA_CERT_PATH, ES_KEY_PATH, ES_CERT_PATH } from './certs';
-export { run, createFailError, createFlagError, combineErrors, isFailError } from './run';
+export { run, createFailError, createFlagError, combineErrors, isFailError, Flags } from './run';
```

Comment:
@spalger I just merged your PR (https://github.com/elastic/kibana/pull/44552) into this PR. Please take a look.